### DISCUSSION TITLE
How to build a CI to integrate YouTube with GitHub

### DISCUSSION THREAD

**Discussion by Khadeeejah** [community/community#37070] [Actions]
I am working on a project that requires translation using crowdin, I manually upload all my transcript from my repo to YouTube, so now I want the CI to  automate the process of uploading the transcript on GitHub to YouTube directly and I guess I am supposed to automate the process with GitHub Api but I am not sure how to go about it.

I would appreciate your feedback thanks

**Comment by ossanna16** [comment]
Hi there @Khadeeejah and welcome to our community! Thank you for asking a great question 🙂

**Comment by Jalalnx** [comment]
great question for sure
